yellow-paper/docs/public-vm/type-structs.md

```diff
@@ -19,7 +19,7 @@ This section lists type definitions relevant to AVM State and Circuit I/O.
 | `callPointer`     | `field`                                | Associates this item with a `TracedContractCall` entry in `worldStateAccessTrace.contractCalls` |
 | `portal`          | `EthAddress`                           |             |
 | `leafIndex`       | `field`                                |             |
-| `msgKey`          | `field`                                | The message key which is also the tree leaf value. |
```

Comment:
Should the `msgKey` here be renamed?

Reply:
This was in the public vm docs, so let that be as @dbanks12 is already in talk around changing things